### ISSUE TITLE
finetuning on specific tag list

### ISSUE THREAD

**Issue by IndigoChildren** [xinyu1205/recognize-anything#185]
Thank you very much for your excellent work. I have some questions to ask you. I need to implement a specific tag list that includes some more common scenes, objects, attributes, and actions. So may I ask if I need to retrain the label embedding when fine-tuning, because I saw that the you froze the label embedding layer in finetune.py. So, what size of dataset do I need to prepare to achieve the fine-tuning effect.
